test/yagna/conftest.py

```diff
@@ -236,21 +241,36 @@ def cancellation_callback() -> Callable[[], None]:
     return lambda: pytest.fail("The runner was cancelled", pytrace=False)
 
 
+@pytest.fixture
+def test_logs_dir(logs_path: Path) -> Path:
```

Comment:
Nitpicking, but functions starting with the prefix `test` are treated as tests by `pytest`, though this one is a fixture. I think it's okay to rename this to `logs_dir` simply (as the docstring provides an explanation).